src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -664,27 +664,51 @@ NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1
 NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1500124, 'local_commands_test_schema', 'ALTER TABLE dist_table ALTER COLUMN c SET NOT NULL;')
 NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1500127, 'local_commands_test_schema', 'ALTER TABLE dist_table ALTER COLUMN c SET NOT NULL;')
 NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1500130, 'local_commands_test_schema', 'ALTER TABLE dist_table ALTER COLUMN c SET NOT NULL;')
-  -- as we create table via remote connections, below SELECT create_distributed_table
-  -- would error out
+  -- as we executed ALTER TABLE commands parallelly, below
+  -- SELECT create_distributed_table would error out
   CREATE TABLE another_dist_table(a int);
```

Comment:
I guess  the fix for this is: https://github.com/citusdata/citus/compare/fix_fkey_rest_bug?expand=1

` ALTER TABLE dist_table ALTER COLUMN c SET NOT NULL;` switches to sequential mode, but still marks as parallel access.

Reply:
@onurctirtir can you rebase this  PR on top of #4141? This should work, if not lets look together

Reply:
After rebase, I had to do this change: https://github.com/citusdata/citus/pull/4139/commits/8ea60ff386c41ad60bf98e88d49b282458acfdc0

I think it's okay that we can execute create_distributed_table after ddl execution on another distributed table ?